microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -121,7 +123,12 @@ When you are done, learn how to work with the simulator through [Apple Developer
 <!-- ------------------------------ -->
 #### Microsoft Edge (EdgeHTML)
 
+<!-- todo: delete section, point to MDN? -->
+
 If you need to test your website or app with Microsoft browsers and don't have the necessary versions of Windows to do so, you can use BrowserStack, which supports testing of many combinations of Microsoft browsers and operating systems both past and present.  For example, you can test all versions of Microsoft Edge (Chromium) from version 80 onwards, and Microsoft Edge (EdgeHTML) versions 15 through 18.  Testing of Microsoft Edge is free on BrowserStack.  For more information, see [Microsoft Edge Browser Testing](https://www.browserstack.com/test-on-microsoft-edge-browser) at BrowserStack.
```

Comment:
is there a relevant current URL?  
check search results: https://www.browserstack.com/search?query=microsoft%20edge

Reply:
fixed by PR 3220 https://github.com/MicrosoftDocs/edge-developer/pull/3220 (Comment out 404 link browserstack.com/test-on-microsoft-edge-browser)